lib/deps_nix.ex

```diff
@@ -124,6 +124,14 @@ defmodule DepsNix do
     end
   end
 
+  defp find_builder_from_path(path) do
+    cond do
+      File.exists?(Path.join(path, "mix.exs")) -> "buildMix"
+      File.exists?(Path.join(path, "rebar.config")) -> "buildRebar3"
+      true -> :unknown
+    end
+  end
```

Comment:
The test suite passes when this function is changed to always return "buildMix". Could we add tests for the other two cases please?

If it's tricky to know where to start or you're pressed for time, I don't mind adding them myself.

Reply:
I’ll look into adding some tests. 👍